docs/composer.md

```diff
@@ -0,0 +1,162 @@
+
+# Composer
+Flarum uses a program called composer[Composer](https://getcomposer.org) to manage its dependencies and extensions.
+You'll need to use composer if you want to:
+
+- Install or update Flarum
+- Install, update, or remove Flarum extensions 
+
+This guide is provided as a brief explanation of Composer. We highly recommend consulting the [official documentation](https://getcomposer.org/doc/00-intro.md) for more information.
+
+::: tip Composer v2
+Historically, composer has caused issues on shared hosting due to huge memory use. In 2020, [Composer v2 was released](https://blog.packagist.com/composer-2-0-is-now-available/) with massive improvements that eliminate these problems. Make sure your server is using composer v2!
+:::
+
+## What is Composer?
+
+>Composer is a tool for dependency management in PHP. It allows you to declare the libraries your project depends on and it will manage (install/update) them for you. — [Composer Introduction]([https://getcomposer.org/doc/00-intro.md](https://getcomposer.org/doc/00-intro.md))
```

Comment:
```suggestion
> Composer is a tool for dependency management in PHP. It allows you to declare the libraries your project depends on and it will manage (install/update) them for you. — [Composer Introduction]([https://getcomposer.org/doc/00-intro.md](https://getcomposer.org/doc/00-intro.md))
```